catalogs/catalog-hadoop/src/test/java/org/apache/gravitino/catalog/hadoop/TestHadoopCatalogOperations.java

```diff
@@ -1493,6 +1494,24 @@ public void testCreateMultipleLocationsWithExceptions() throws IOException {
                       null));
       Assertions.assertEquals("Location name must not be blank", exception.getMessage());
 
+      // empty location in catalog location
```

Comment:
could you plz also test the cases that:
1. empty location in the schema location
2. empty fileset storage location